Errors with noUnusedLocals

In my project, we use stricter TypeScript option i.e.`noUnusedLocals`
I saw compilation error from ngx-clipboard.

```
src/clipboard.directive.ts(3,5): error TS6133: 'ElementRef' is declared but never used.
src/clipboard.service.ts(1,60): error TS6133: 'InjectionToken' is declared but never used.
```

It seems easy to fix, so I tried to fix it.
But it was difficult.

After I deleted importing InjectToken from src/clipboard.service.ts, I met a new error.
```
src/clipboard.service.ts(105,14): error TS4023: Exported variable 'CLIPBOARD_SERVICE_PROVIDER' has or is using name 'InjectionToken' from external module "/work/ngx-clipboard/node_modules/@angular/core/src/di/injection_token" but cannot be named.
```

That error seems https://github.com/Microsoft/TypeScript/issues/5711

I really want to solve this problem, does anyone have good ideas?

@maxisam, with the latest version (8.1.1) I got only one error:
````
ERROR in /home/sandro/chatz-website/node_modules/ngx-clipboard/src/clipboard.service.ts
(1,60): error TS6133: 'InjectionToken' is declared but never used.
````
````
import { Inject, Injectable, Optional, Renderer, SkipSelf, InjectionToken } from '@angular/core';
````
This can be easily solved by removing ````InjectionToken```` since its not used in this file.

--- a/src/clipboard.service.ts
+++ b/src/clipboard.service.ts
@@ -103,7 +103,11 @@ export function CLIPBOARD_SERVICE_PROVIDER_FACTORY(doc: Document, win: Window, p
 }
 
 export const CLIPBOARD_SERVICE_PROVIDER = {
-    deps: [DOCUMENT, WINDOW, [new Optional(), new SkipSelf(), ClipboardService]],
+    deps: [
+        DOCUMENT as InjectionToken<Document>,
+        WINDOW as InjectionToken<Document>,
+        [new Optional(), new SkipSelf(), ClipboardService]
+    ],
     provide: ClipboardService,
     useFactory: CLIPBOARD_SERVICE_PROVIDER_FACTORY
 };

